src/services/redisSessionStorage.ts

```diff
@@ -40,7 +41,7 @@ export default class RedisSessionStorage extends RedisStorageUtils
   constructor(
     private readonly redisClient: redis.RedisClient,
     private readonly tokenDurationSecs: number,
-    private readonly allowMultipleSessions: boolean
+    private readonly allowMultipleSessions: ALLOW_MULTIPLE_SESSIONS_OPTION
```

Comment:
the boolean was ok (RedisServiceOptions would be better but I'd leave it out of this PR)

Reply:
Fixed in aea043b